index.js

```diff
@@ -0,0 +1,13 @@
+module.exports = {
```

Comment:
That's a good idea... I was thinking about doing it that way, but then had a feeling I shouldn't. 

In retrospect, I think that feeling was me remembering dynamic `require`'s don't work in Browserify. But, this isn't for Browserify, so that reticence is probably just superstition.